Sources/swiftML/Helpers.swift

```diff
@@ -0,0 +1,87 @@
+import TensorFlow
+
+/// The generalized inverse of a matrix using its singular-value decomposition (SVD).
+///
+/// Returns the (Moore-Penrose) pseudo-inverse of a matrix.
+///
+/// - Parameter data: The input matrix to be pseudo-inverse.
+/// - Returns: Pseudo-inverse matrix.
+public func matrixPseudoInverse(_ data: Tensor<Float>) -> Tensor<Float> {
+    let svd = Raw.svd(data)
+    let diag = Raw.diag(diagonal: svd.s)
+    return matmul(matmul(svd.v, Raw.matrixInverse(diag)), svd.u.transposed())
+}
+
+/// Returns Sign correction to ensure deterministic output from SVD.
+///
+/// - Parameters
+///   - u: The tensor containing of left singular vectors for each matrix.
+///   - v: The tensor containing of right singular vectors for each matrix.
+/// - Returns: Sign correction to ensure deterministic output.
+public func svdFlip(
+    u: Tensor<Double>,
+    v: Tensor<Double>,
+    uBasedDecision: Bool = true
+) -> (Tensor<Double>, Tensor<Double>) {
+  
+    var U = u
```

Comment:
Please create mutable local variables named `u` and `v` instead of `U` and `V`. Swift supports this kind of aliasing.
```suggestion
    var u = u
    var v = v
```